my_style.css

```diff
@@ -0,0 +1,82 @@
+body{
+	
+background-color: #FFEBCD;
+	
+}
+
+
+
+a:link{
+	color: red; 
+}
+a:visited{
+	color: #191970;
+}
+a:hover{
+	color: green; 
+}
+
+nav a{
+	 float: left; 
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none; 
+  font-size: 17px;
+  background-color: #FF8C00;
+}
+
+nav a:hover{
+	background-color: #ddd; 
+	color: blue; 
+}
+nav a.active{
+	background-color: #d2691e;
+	color: #fff8dc;
+}
+
+.site-footer{
```

Comment:
A footer is already a very precise section that you will always use as a footer, so you don't need to add a class to it. You could just style directly the footer.